.github/workflows/manual_tests.yml

```diff
@@ -0,0 +1,149 @@
+# Tests artifacts
+name: manual_tests
+on:
+  # triggered manually
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Garden Linux Version"
+        type: string
+        default: now
+      build:
+        description: "Build for version 'today' instead of downloading version from S3"
```

Comment:
```suggestion
        description: "Build for version specified in version input parameter, instead of downloading version from S3"
```

Reply:
I changed some more descriptions while working at it.